index.html

```diff
@@ -3,6 +3,11 @@
     <head>
         <title>Books Shop</title>
         <link rel="stylesheet" href="styles.css">
+        <link rel="stylesheet" href="normalize.css">
```

Comment:
```suggestion
        <link rel="stylesheet" href="normalize.css">
        <link rel="stylesheet" href="styles.css">
```
Каскад работает так, что запись которая расположена ниже перетирает все вышестоящие. Обычно первым используют normalize.css - он убирает стили браузера, а уже затем собственные стили. Так как если селекторы с одинаковым весом расположены в обоих файлах, то normalaize перетрет наше правило.